dicom_validator/tests/spec_reader/test_part3_reader.py

```diff
@@ -246,15 +246,13 @@ def test_graphic_annotation_sequence(self, reader):
         assert "(0070,0001)" in description
         assert "items" in description["(0070,0001)"]
 
-        graphic_object_sequence = "(0070,0009)"
-        compound_graphic_sequence = "(0070,0209)"
-        for sequence_tag in (graphic_object_sequence, compound_graphic_sequence):
```

Comment:
(0070,0209) actually has another condition, so I removed in from the test. This is a problem with the SPECIAL_CASES mechnism, as it would change the condition for all occurrences of a tag.

Reply:
Ah, my mistake, I've looked it up briefly and did not check the tags carefully, sorry.